packages/components-native/src/Typography/Typography.tsx

```diff
@@ -335,30 +335,90 @@ export type TextColor =
   | "tealDark"
   | "indigoDark"
   | "navy"
-  | "text"
+
+  // Typography
   | "heading"
+  | "text"
   | "textSecondary"
   | "textReverse"
   | "textReverseSecondary"
-  | "interactive"
-  | "interactiveSubtle"
-  | "destructive"
-  | "learning"
-  | "subtle"
-  | "onPrimary";
 
-export type TextVariation =
+  // Statuses
+  | "inactive"
+  | "inactiveSurface"
+  | "inactiveOnSurface"
+  | "critical"
+  | "criticalSurface"
+  | "criticalOnSurface"
+  | "warning"
+  | "warningSurface"
+  | "warningOnSurface"
+  | "informative"
+  | "informativeSurface"
+  | "informativeOnSurface"
   | "success"
+  | "successSurface"
+  | "successOnSurface"
+
+  // Interactions
+  | "interactiveHover"
+  | "interactiveSubtleHover"
+  | "destructiveHover"
+  | "disabledSecondary"
+  | "surface"
```

Comment:
Bit of a nitpick at this point, but since we decided to leave `surface` out in [web](https://github.com/GetJobber/atlantis/pull/2030/files) I think we should do the same here, and not include it.
```suggestion

```

Reply:
Ah good call, this should be removed